packages/worker/src/tests/api/environment.ts

```diff
@@ -9,6 +9,7 @@ export class EnvironmentAPI extends TestAPI {
   getEnvironment = () => {
     return this.request
       .get(`/api/system/environment`)
+      .set(this.config.defaultHeaders())
```

Comment:
Out of interest - why is this needed now? The environment endpoint usually can be called publicly I believe.

Reply:
Ah, without a tenantId it will now fail because the tenantId is needed to check the feature flag. Will this be a problem in practice?

Reply:
It would be worth checking if its a problem - if you publish an app with a public screen, then try to go to that screen URL in the app I would suspect you'll get some error messages if the environment endpoint now requires it.

We will probably have to assume feature flags are disabled for public apps/pages until the feature is fully released, since there is no real way to confirm the tenant ID.